utils/readers.ts

```diff
@@ -28,3 +46,24 @@ export const getPopDocumentReader = (
     })),
     TE.chainW(TE.fromOption(() => ({ kind: ErrorKind.NotFound as const })))
   );
+
+// IMPLEMENTATIONS
```

Comment:
ditto

Reply:
Done in [80a493d](https://github.com/pagopa/io-functions-lollipop/pull/26/commits/80a493db6bd5b57663e3f88ef3461acf83972ec1)